src/main/java/dinkplugin/notifiers/PetNotifier.java

```diff
@@ -215,55 +236,294 @@ private static class ParseResult {
+                    return Stream.of("", " Challenge Mode")
+                        .map(suffix -> "Chambers of Xeric" + suffix)
+                        .map(event -> kcService.getKillCount(LootRecordType.EVENT, event))
+                        .filter(Objects::nonNull)
+                        .reduce(Integer::sum)
+                        .orElse(null);
+                }
+            }),
+            entry("Pet chaos elemental", new MultiKcSource("Chaos Elemental", 1.0 / 300, "Chaos Fanatic", 1.0 / 1_000)),
+            entry("Pet dagannoth prime", new KcSource("Dagannoth Prime", 1.0 / 5_000)),
+            entry("Pet dagannoth rex", new KcSource("Dagannoth Rex", 1.0 / 5_000)),
+            entry("Pet dagannoth supreme", new KcSource("Dagannoth Supreme", 1.0 / 5_000)),
+            entry("Pet dark core", new KcSource("Corporeal Beast", 1.0 / 5_000)),
+            entry("Pet general graardor", new KcSource("General Graardor", 1.0 / 5_000)),
+            entry("Pet k'ril tsutsaroth", new KcSource("K'ril Tsutsaroth", 1.0 / 5_000)),
+            entry("Pet kraken", new KcSource("Kraken", 1.0 / 5_000)),
+            entry("Pet penance queen", new Source() {
+                @Override
+                Double getProbability(Client client, KillCountService kcService) {
+                    return 1.0 / 1_000;
+                }
+
+                @Override
+                Integer estimateActions(Client client, KillCountService kcService) {
+                    return client.getVarbitValue(Varbits.BA_GC);
+                }
+            }),
+            entry("Pet smoke devil", new KcSource("Thermonuclear smoke devil", 1.0 / 3_000)),
+            entry("Pet snakeling", new KcSource("Zulrah", 1.0 / 4_000)),
+            entry("Pet zilyana", new KcSource("Commander Zilyana", 1.0 / 5_000)),
+            entry("Phoenix", new KcSource("Wintertodt", 1.0 / 5_000)),
+            entry("Prince black dragon", new KcSource("King Black Dragon", 1.0 / 3_000)),
+            entry("Rift guardian", new SkillSource(Skill.RUNECRAFT, 1_795_758, client -> client.getSkillExperience(Skill.RUNECRAFT) / 10)), // lava runes
+            entry("Rock golem", new SkillSource(Skill.MINING, 211_886, client -> client.getSkillExperience(Skill.MINING) / 65)), // gemstones
+            entry("Rocky", new SkillSource(Skill.THIEVING, 36_490, client -> client.getSkillExperience(Skill.THIEVING) / 42)), // stalls
+            entry("Scorpia's offspring", new KcSource("Scorpia", 1 / 2015.75)),
+            entry("Scurry", new KcSource("Scurrius", 1.0 / 3_000)),
+            entry("Skotos", new KcSource("Skotizo", 1.0 / 65)),
+            entry("Smolcano", new KcSource("Zalcano", 1.0 / 2_250)),
+            entry("Sraracha", new KcSource("Sarachnis", 1.0 / 3_000)),
+            entry("Tangleroot", new SkillSource(Skill.FARMING, 7_500, client -> client.getSkillExperience(Skill.FARMING) / 119)), // mushrooms
+            entry("Tiny tempor", new KcSource("Tempoross", 1.0 / 8_000)),
```

Comment:
Might be good to double check this value, to see if we can get amount of loots

Reply:
good idea - would work if loot tracker is enabled https://github.com/runelite/runelite/blob/99a66ccd693d3190c55c14d89b94092de63b01ee/runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java#L273